<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGraphicsView" name="imageView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="inputImageViewAdd">
            <property name="text">
             <string>Add more image</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGraphicsView" name="outputImageView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="outputImageViewPlay">
            <property name="text">
             <string>Play</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="outputImageViewPause">
            <property name="text">
             <string>Pause</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="outputImageViewStop">
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="file_Exit"/>
   </widget>
   <widget class="QMenu" name="menuEffects">
    <property name="title">
     <string>Concat</string>
    </property>
    <addaction name="hConcat"/>
    <addaction name="vConcat"/>
    <addaction name="triConcat"/>
   </widget>
   <widget class="QMenu" name="menuAnimation">
    <property name="title">
     <string>Animation</string>
    </property>
    <widget class="QMenu" name="hAnimation">
     <property name="title">
      <string>H Animation</string>
     </property>
     <addaction name="hAnimation_Left_In"/>
     <addaction name="hAnimation_Right_In"/>
     <addaction name="hAnimation_Left_Out"/>
     <addaction name="hAnimation_Right_Out"/>
    </widget>
    <widget class="QMenu" name="vAnimation">
     <property name="title">
      <string>V Animation</string>
     </property>
     <addaction name="vAnimation_Top_In"/>
     <addaction name="vAnimation_Bottom_In"/>
     <addaction name="vAnimation_Top_Out"/>
     <addaction name="vAnimation_Bottom_Out"/>
    </widget>
    <addaction name="hAnimation"/>
    <addaction name="vAnimation"/>
   </widget>
   <widget class="QMenu" name="menuTransition">
    <property name="title">
     <string>Transition</string>
    </property>
    <widget class="QMenu" name="rectTransfer">
     <property name="title">
      <string>Rect Transfer</string>
     </property>
     <addaction name="rectTransfer_Top_Left_To"/>
     <addaction name="rectTransfer_Top_Right_To"/>
     <addaction name="rectTransfer_Bottom_Left_To"/>
     <addaction name="rectTransfer_Bottom_Right_To"/>
     <addaction name="rectTransfer_Border_To_Center"/>
     <addaction name="rectTransfer_Center_To_Border"/>
     <addaction name="rectTransfer_Left_To_Right"/>
     <addaction name="rectTransfer_Right_To_Left"/>
     <addaction name="rectTransfer_Top_To_Bottom"/>
     <addaction name="rectTransfer_Bottom_To_Top"/>
    </widget>
    <addaction name="transition"/>
    <addaction name="rectTransfer"/>
   </widget>
   <widget class="QMenu" name="menuFeatures">
    <property name="title">
     <string>Features</string>
    </property>
    <addaction name="features_SLIC"/>
    <addaction name="features_SIFT"/>
    <addaction name="features_ASLIC"/>
   </widget>
   <widget class="QMenu" name="menuMisc">
    <property name="title">
     <string>Misc</string>
    </property>
    <addaction name="misc_Super_Resolution"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEffects"/>
   <addaction name="menuAnimation"/>
   <addaction name="menuTransition"/>
   <addaction name="menuFeatures"/>
   <addaction name="menuMisc"/>
  </widget>
  <action name="hConcat">
   <property name="text">
    <string>hConcat</string>
   </property>
  </action>
  <action name="vConcat">
   <property name="text">
    <string>vConcat</string>
   </property>
  </action>
  <action name="triConcat">
   <property name="text">
    <string>triConcat</string>
   </property>
  </action>
  <action name="hAnimation_Left_In">
   <property name="text">
    <string>Left In</string>
   </property>
  </action>
  <action name="hAnimation_Right_In">
   <property name="text">
    <string>Right In</string>
   </property>
  </action>
  <action name="hAnimation_Left_Out">
   <property name="text">
    <string>Left Out</string>
   </property>
  </action>
  <action name="hAnimation_Right_Out">
   <property name="text">
    <string>Right Out</string>
   </property>
  </action>
  <action name="vAnimation_Top_In">
   <property name="text">
    <string>Up In</string>
   </property>
  </action>
  <action name="vAnimation_Bottom_In">
   <property name="text">
    <string>Bottom In</string>
   </property>
  </action>
  <action name="vAnimation_Top_Out">
   <property name="text">
    <string>Up Out</string>
   </property>
  </action>
  <action name="vAnimation_Bottom_Out">
   <property name="text">
    <string>Bottom Out</string>
   </property>
  </action>
  <action name="transition">
   <property name="text">
    <string>Transparent</string>
   </property>
  </action>
  <action name="features_SLIC">
   <property name="text">
    <string>SLIC</string>
   </property>
  </action>
  <action name="features_SIFT">
   <property name="text">
    <string>SIFT</string>
   </property>
  </action>
  <action name="features_ASLIC">
   <property name="text">
    <string>ASLIC</string>
   </property>
  </action>
  <action name="misc_Super_Resolution">
   <property name="text">
    <string>Super resolution</string>
   </property>
  </action>
  <action name="file_Exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="rectTransfer_Top_Left_To">
   <property name="text">
    <string>Top_Left_To</string>
   </property>
  </action>
  <action name="rectTransfer_Top_Right_To">
   <property name="text">
    <string>Top_Right_To</string>
   </property>
  </action>
  <action name="rectTransfer_Bottom_Left_To">
   <property name="text">
    <string>Bottom_Left_To</string>
   </property>
  </action>
  <action name="rectTransfer_Bottom_Right_To">
   <property name="text">
    <string>Bottom_Right_To</string>
   </property>
  </action>
  <action name="rectTransfer_Border_To_Center">
   <property name="text">
    <string>Center_To_Border</string>
   </property>
  </action>
  <action name="rectTransfer_Center_To_Border">
   <property name="text">
    <string>Border_To_Center</string>
   </property>
  </action>
  <action name="rectTransfer_Left_To_Right">
   <property name="text">
    <string>Left_To_Right</string>
   </property>
  </action>
  <action name="rectTransfer_Right_To_Left">
   <property name="text">
    <string>Right_To_Left</string>
   </property>
  </action>
  <action name="rectTransfer_Top_To_Bottom">
   <property name="text">
    <string>Top_To_Bottom</string>
   </property>
  </action>
  <action name="rectTransfer_Bottom_To_Top">
   <property name="text">
    <string>Bottom_To_Top</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
